src/Microsoft.ML.Data/Transforms/NormalizeColumn.cs

```diff
@@ -283,10 +281,8 @@ public static IDataTransform Create(IHostEnvironment env, MinMaxArguments args,
             return normalizer.Fit(input).MakeDataTransform(input);
         }
 
-        /// <summary>
-        /// Public create method corresponding to SignatureDataTransform.
-        /// </summary>
-        public static IDataTransform Create(IHostEnvironment env, MeanVarArguments args, IDataView input)
+        // Factory method corresponding to SignatureDataTransform.
+        internal static IDataTransform Create(IHostEnvironment env, MeanVarArguments args, IDataView input)
```

Comment:
>internal [](start = 8, length = 8)

If the comment is correct, then it should suffice for it to be `private`, right? So is the comment incorrect?

Reply:
Everytime we have Entrypoint for transform I have to mark this method as internal.
As alternative I can refactor conversion of argument class to ColumnInfo array and use it here and in entrypoint, but oh, boy, it would be lot of changes across codebase.

---
In reply to: [245457063](https://github.com/dotnet/machinelearning/pull/2031#discussion_r245457063) [](ancestors = 245457063)

Reply:
Oh entrypoints. Fair enough.

---
In reply to: [245457335](https://github.com/dotnet/machinelearning/pull/2031#discussion_r245457335) [](ancestors = 245457335,245457063)